### PR TITLE
StartUp improvements

### PR DESCRIPTION
**Pull request by FinnStutzenstein**

### CODE REVIEW

File: openslides/core/static/css/app.css [hunk 1]
```diff
@@ -130,23 +130,36 @@ img {
...
+#spinner-container div {
+    width: 100%;
+    position: absolute;
+    bottom: 0;
     font-size: 56px;
     text-align: center;
+    color: #317796;
 }
-#startup-overlay > div {
+/*#startup-overlay > div {
```
Comment (normanjaeckel) [inline openslides/core/static/css/app.css]: Please remove these lines.

File: openslides/core/static/js/core/start.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ angular.module('OpenSlidesApp.core.start', [])
     'Group',
     'mainMenu',
     function($http, $rootScope, $state, autoupdate, operator, Group, mainMenu) {
-        $rootScope.startupWaitingEnabled = true;
+        $rootScope.openslidesBootstrap = true;
```
Comment (normanjaeckel) [inline openslides/core/static/js/core/start.js]: Maybe call it `openslidesBootstrapReady` or `openslidesBootstrapDone`?

**Comment by normanjaeckel** [comment]
Some proposals.

File: openslides/core/static/css/app.css (hunk 1 above)
Comment (FinnStutzenstein) [inline openslides/core/static/css/app.css]: Done.

File: openslides/core/static/js/core/start.js (hunk 2 above)
Comment (FinnStutzenstein) [inline openslides/core/static/js/core/start.js]: Now its `openslidesBootstrapDone` and so the logic is now inverted.

**Comment by FinnStutzenstein** [comment]
@normanjaeckel Done and rebased.

**Pull request merged**